packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1025,6 +1046,25 @@ export class SnapController extends BaseController<
     Object.values(this.state?.snaps ?? {}).forEach((snap) =>
       this.#setupRuntime(snap.id),
     );
+
+    this.#trackSnapExport = throttleTracking(
```

Comment:
We should check with Christian if we want to track exports for preinstalled Snaps too

Reply:
For other metrics, we don't always track preinstalled Snaps because they have a lot of usage and spam the metrics a bit.

E.g. https://github.com/MetaMask/metamask-extension/blob/main/app/scripts/metamask-controller.js#L3048

Reply:
Some of our preinstalled Snaps run cronjobs every minute, so our metrics would probably be flooded with that 😄